libbeat/cfgfile/glob_manager.go

```diff
@@ -182,3 +189,40 @@ func (g *GlobManager) Disable(name string) error {
 
 	return errors.Errorf("module %s not found", name)
 }
+
+// DisplayName returns the config file's display name
+func (f *CfgFile) DisplayName() string {
```

Comment:
Why not call it `String`? I think it is more common to use functions named `String` to get the string representation of a type.

Reply:
@kvch Made this change in 9ba27c7.